adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -68,6 +67,7 @@ public void export(final NonStandardHeaderNotice toExport) throws IOException {
                 .setType(GtfsValidationOutputProto.GtfsProblem.Type.TYPE_UNKNOWN_COLUMN)
                 .setSeverity(GtfsValidationOutputProto.GtfsProblem.Severity.SUSPICIOUS_WARNING)
                 .setAltEntityValue(toExport.getExtraHeader())
+                .clearAltEntityId()
```

Comment:
Good catch! As we said together, replace this by a call to `clear()` at the beginning of the chain to clear the reused internal builder